sae_lens/load_model.py

```diff
@@ -40,5 +47,130 @@ def load_model(
                 model_name, device=cast(Any, device), **model_from_pretrained_kwargs
             ),
         )
+    elif model_class_name == "AutoModelForCausalLM":
```

Comment:
I don't have a strong opinion on this, if we can have a linting rule set up to enforce a style here I'm happy with that. the .pylintrc is likely legacy, so I'll delete that. I'd support moving to [Ruff](https://docs.astral.sh/ruff/), since that supports everything from pylint, flake8, pyflakes, isort, black, etc, is a lot faster, and seems to be what the industry is moving towards. I'll open an issue for this.